src/stylesheets/components/_sidenav.scss

```diff
@@ -1,17 +1,17 @@
 .usa-sidenav-list {
   @include usa-sidenav-list;
```

Comment:
I feel like we should something like the following to `settings-typography`

```
$theme-font-family-sidenav:         'ui' !default;
```

...then typeset this component with `@include typeset($theme-font-family-sidenav, null, 3)`. This could take away the need to add explicit line height deeper in the component? It also isolates the sidenav from the body font choice, which I think could be useful.

Then use `$theme-font-family-sidenav` in subsequent font functions instead of `'body'`

Reply:
Done in f042eab.